router/batchrouter/asyncdestinationmanager/bing-ads/bingads.go

```diff
@@ -290,7 +377,48 @@ func NewManager(destination *backendconfig.DestinationT, backendConfig backendco
 		panic(fmt.Errorf("Error in unmarshalling destination config: %v", err))
 	}
 	oauthClient := oauth.NewOAuthErrorHandler(backendConfig)
+	sessionConfig := bingads.SessionConfig{
+		AccountId:  destConfig.CustomerAccountId,
+		CustomerId: destConfig.CustomerId,
+		HTTPClient: http.DefaultClient,
+		AccessTokenGenerator: func() (string, string, error) {
```

Comment:
Define this function somewhere else as we need to write tests for this

Reply:
@ItsSudip